aptos-move/framework/supra-framework/sources/multisig_account.move

```diff
@@ -253,6 +264,12 @@ module supra_framework::multisig_account {
         new_metadata: SimpleMap<String, vector<u8>>,
     }
 
+    /// Event emitted when a transaction's timeout duration is updated.
+    struct TimeoutDurationUpdatedEvent has drop, store {
+        old_timeout_duration: u64,
+        new_timeout_duration: u64,
```

Comment:
Also add the `address` of the multisig account as part of the event.

Reply:
Done.